packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```diff
@@ -54,12 +54,18 @@ export default function WorkflowDetailsPage(props: Props) {
         if (teamId && teamId !== group.teamId) return options;
         return [
           ...options,
-          ...group.eventTypes.map((eventType) => ({
-            value: String(eventType.id),
-            label: `${eventType.title} ${
-              eventType.children && eventType.children.length ? `(+${eventType.children.length})` : ``
-            }`,
-          })),
+          ...group.eventTypes
+            .filter((evType) =>
+              evType.schedulingType === "MANAGED"
+                ? evType.metadata?.managedEventConfig?.unlockedFields?.workflows !== undefined
+                : true
+            )
```

Comment:
Added this filter to avoid having members select managed event types as the target for workflows when workflows are locked.